### ISSUE TITLE
How can I get desired depart value for vehicles by using traci interface?

### ISSUE THREAD

**Issue by Summer142857** [eclipse/sumo#7693]

**Comment by behrisch** [comment]
Currently you cannot, I am leaving this open as a feature request.

**Comment by Matyxus** [comment]
Hello, i would like to ask, if this feature is still being worked on ? I really need it for my project (planning vehicle routes with planners).

**Comment by namdre** [comment]
Should be something like `vehicle.getDepart`, `vehicle.setDepart`.

Until this gets implemented, consider using sumolib to parse the departure times from the xml input.
```
for veh in sumolib.xml.parse(routefile, "vehicle")
    time = sumolib.miscutils.parseTime(veh.depart)
```

**Comment by behrisch** [comment]
will be solved in #7891
